CMakeLists.txt

```diff
@@ -63,6 +66,8 @@ find_package(UUID REQUIRED)
 file(GLOB SPD_SOURCES "include/spdlog/*")
 
 add_subdirectory(thirdparty/yaml-cpp-yaml-cpp-0.5.3)
+set(CIVETWEB_ENABLE_CXX ON CACHE BOOL "Enable civet C++ library")
```

Comment:
Could we download and then build  civet using ExternalProject instead of bundling it?